wgpu/tests/example_wgsl.rs

```diff
@@ -1,6 +1,7 @@
 use naga::{front::wgsl, valid::Validator};
```

Comment:
why are we renaming this example? is there anything wrong with a dash there?

Reply:
We're renaming this because it's now a submodule of the root integration test and dash is an invalid module name.